lib/src/postgres_target.rs

```diff
@@ -502,7 +439,7 @@ pub async fn init(
                     let client =
                         update_postgres_client(&mut client_opt, &postgres_slot, &config).await;
                     if let Err(err) = slots_processing
-                        .process(client, &update, &preprocessing)
+                        .process(client, &update, write_batch, &preprocessing)
```

Comment:
no I don't think we want the write batch here. we're already writing the account updates above in line 328-385.